### ISSUE TITLE
Submissions: Tiny AES

### ISSUE THREAD

**Issue by Photosounder** [r-lyeh/single_file_libs#134]
Two-file public domain AES encryption/decryption in C
https://github.com/kokke/tiny-AES-c

**Comment by HSchmale16** [comment]
Crypto is one of those things you don't want to do yourself except as a
learning experience. This is mildly dangerous. Does the author have a
strong background in cryptography?

**Comment by Photosounder** [comment]
Good point which I hadn't considered, however it could still be good for low security applications like decoding a public stream. I found out about Tiny AES from [hlsdl](https://github.com/samsamsam-iptvplayer/hlsdl) which uses Tiny AES to decode streams, perhaps that's why I never thought of security as a problem in such a situation. So this couldn't replace a well security-tested implementation and you wouldn't want to put it in a server, but other than that it could probably be suitable for many things. Either way it definitely suits the philosophy and format of this catalogue of tiny libraries.

**Comment by r-lyeh** [comment]
Closing ticket. Link(s) should be already in the list. Please double-check.
